packages/elements/src/components/ino-input-file/ino-input-file.tsx

```diff
@@ -50,6 +66,57 @@ export class InputFile implements ComponentInterface {
     files: object[];
   }>;
 
+  componentDidLoad(): void {
+    this.configureDragAndDrop();
+  }
+
+  private configureDragAndDrop(): void {
+    if (this.browserSupportsDragAndDrop()) {
+      const box = this.el.querySelector('.ino-input-file__dnd');
+
+      this.addEventListeners(
+        this.el,
+        'drag dragstart dragend dragover dragenter dragleave drop',
+        (e) => {
+          e.preventDefault();
+          e.stopPropagation();
+        }
+      );
+      this.addEventListeners(
+        this.el,
+        'dragover dragenter',
+        () => {
+          box.classList.add('ino-input-file__dnd-dragover');
+        }
+      );
+      this.addEventListeners(
+        this.el,
+        'dragend dragleave drop',
+        () => box.classList.remove('ino-input-file__dnd-dragover')
+      );
+      this.el.addEventListener('drop', (e) => {
+        if (this.disabled) {
+          return;
+        }
+        console.log('Drop');
+        const files = e.dataTransfer.files as FileList;
```

Comment:
I think we could remove the as FileList type conversion

Reply:
✅